hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestCompactionState.java

```diff
@@ -164,10 +166,10 @@ private void compaction(final String tableName, final int flushes,
       long curt = System.currentTimeMillis();
       long waitTime = 5000;
       long endt = curt + waitTime;
-      CompactionState state = admin.getCompactionState(table);
+      CompactionState state = master.getCompactionState(table);
```

Comment:
I think this test is for testing the HBaseAdmin.getCompactionState so we should not change to use master?

Reply:
Make sense, will fix later, thanks.